.gitignore

```diff
@@ -0,0 +1,6 @@
+.vscode
```

Comment:
you're right..... i'm a bit confused as well. i'll troubleshoot and see where the problem is coming from.
Thank you!

Reply:
Maybe easy fix will be to push everything as it is and after remove folder from github manually
I believe it will never appears after it because of gitignore

Reply:
Ohkay, i'm doing that now